cmake/Fields2CoverConfig.cmake.in

```diff
@@ -4,4 +4,9 @@ include("${CMAKE_CURRENT_LIST_DIR}/Fields2Cover-Targets.cmake" )
 
 include(CMakeFindDependencyMacro)
 find_dependency(GDAL 3.0 REQUIRED)
-find_dependency(ortools REQUIRED)
+
+list(INSERT CMAKE_PREFIX_PATH 0 "${PACKAGE_PREFIX_DIR}/opt/ortools_vendor")
+list(INSERT CMAKE_PREFIX_PATH 1 "${PACKAGE_PREFIX_DIR}/opt/f2c_ortools")
```

Comment:
This *should* allow the ortools to be found in the cases where they are provided by USE_ORTOOLS_VENDOR or USE_ORTOOLS_RELEASE.  Preference for ortools_vendor version.